.cargo/config.toml

```diff
@@ -1,3 +1,6 @@
 [build]
 rustdoc = "rustdoc"
 rustdocflags = ["--show-type-layout", "-Z", "unstable-options"]
+
+[unstable]
+sparse-registry = true
```

Comment:
what's this

Reply:
just switches the cargo indexing to use `http` rather than their default git one, it makes ci builds and package fetching a lot faster.

Reply:
https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#sparse-registry

Reply:
Are we using any of these?

Reply:
well no it's just how packages are fetched

Reply:
Alright then